### DISCUSSION TITLE
Local Git Provider

### DISCUSSION THREAD

**Discussion by stramel** [six7/figma-tokens#1310] [Ideas]
It would be nice to be able to support a standard git repo directly to catch all cases that are not officially supported via providers. This could be something simple using an npm package like `simple-git`

**Comment by six7** [comment]
I dont think we can sync to a file as we're just living in a figma iframe. Happy to be proven wrong on that one though! We built file import/export to work around that problem, but having a way to use local git would be preferrable.

**Reply by stramel** [reply]
Seems that Figma is not currently allowing access inside their iframe to the FileSystemAccess API.
